environment.yml

```diff
@@ -30,7 +30,7 @@ dependencies:
   - pandas>=2
   - pymongo
   - seaborn
-  - ctapipe_io_lst=0.21.1
+  - ctapipe_io_lst=0.22.0
```

Comment:
I'd suggest to match the definition in setup.cfg. Allow any 0.22.x version.

So instead of `=0.22.0`, I'd put `= 0.22`, which allows newer versions once we release them